cli/Sources/TuistKit/Commands/Auth/RefreshTokenCommand.swift

```diff
@@ -0,0 +1,32 @@
+import ArgumentParser
+import Foundation
+
+struct RefreshTokenCommand: AsyncParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "refresh-token",
+            _superCommandName: "auth",
+            abstract: "Refreshes the token for a particular URL",
+            shouldDisplay: false
+        )
+    }
+
+    @Option(
+        name: .customLong("server-url"),
+        help: "The URL of the server the token is being refreshed for.",
+        envKey: .authRefreshTokenServerURL
+    )
+    var serverURL: String
```

Comment:
nit: I know this one is hidden but I'd still make this one an argument (`tuist auth refresh-token tuist.dev`)